### PR TITLE
add a truth

### PR DESCRIPTION
**Pull request by Gingertonic**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -8,5 +8,7 @@
 </head>
 <body>
     <h1>Coming soon...</h1>
+
+    <p>Cats are the best</p>
```
Comment (Cazbean) [inline index.html]: Cats are in fact NOT the best

**Review by Cazbean — CHANGES_REQUESTED** [review]
Quick as poss thanks!

File: index.html (hunk 1 above)
Comment (Gingertonic) [reply index.html]: Hey @Cazbean what should I put instead?

Comment (Cazbean) [reply index.html]: Please put that dogs are the best.

File: index.html [hunk 2]
```diff
@@ -9,6 +9,6 @@
 <body>
     <h1>Coming soon...</h1>
 
-    <p>Cats are the best</p>
+    <p>Dogs are the best. But we have only cats...</p>
```
Comment (Cazbean) [inline index.html:12]: Thank you!

**Review by Cazbean — COMMENTED** [review]
All good

**Review by Cazbean — APPROVED** [review]

**Pull request merged**